README.md

```diff
@@ -19,8 +19,7 @@ Godot is completely free and open source under the very permissive MIT license.
 No strings attached, no royalties, nothing. The users' games are theirs, down
 to the last line of engine code. Godot's development is fully independent and
 community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the Software Freedom Conservancy
-not-for-profit.
+expectations. The Software Freedom Conservancy not-for-profit supports it.
```

Comment:
I'm not a native speaker, but this reads worse to me. The previous sentence makes it clear that SFC is Godot's support structure. The new sentence sounds like SFC happens to be a supporter of Godot, but there might be others and it's not clear why it's relevant.

Reply:
CC @cbscribe @NathanLovato

Reply:
The previous sentence is not ideal due to passive voice, but I don't know if this is any better, since it does read weirdly.

Perhaps we can change around the verb so that the action is Godot receiving rather than SFC giving: "Godot receives support from the Software Freedom Conservancy not-for-profit" or similar.